docs/ReleaseHistory.md

```diff
@@ -11,6 +11,9 @@
 - FPS => False positive reduction in static analysis.
 - FNS => False negative reduction in static analysis.
 
+# UNRELEASED
+- NEW: Sort properties by name in GeneratedRegexPatterns/*.json.
```

Comment:
I'm on the fence regarding whether this change is worth documenting at all.  Or whether it's a `NEW` label. We might have labeled this as clean-up, fundamentals, work in a separate project. `FUN`. SEV4, honestly, just leave it alone.